andela/bp-esa-frontend
# feat(Automation Modal): Implement automation modal

#### What does this PR do?
It creates automation modal cards

#### Description of Task to be completed?

- Remove the previous implementation of the automation modals
- Create the automation cards according to the new design mockups
- Adds tests

#### How should this be manually tested?
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-164590158-implement-automation-card` branch which contains the changes.
- Use $ yarn start to start the application
- Visit http://localhost:3000/.
- On the Report page click the information icon at the extreme right of each automation table row. A modal with the automation details will appear.

#### What are the relevant pivotal tracker stories?
[#164536900 ](https://www.pivotaltracker.com/story/show/164590158)

#### Screenshots

- **Slack Automation**
<img width="1440" alt="Screen Shot 2019-03-28 at 12 12 24" src="https://user-images.githubusercontent.com/16057093/55145087-236d1e80-5153-11e9-9b0c-2f00500bd6ad.png">

- **Email Automation**
<img width="1440" alt="Screen Shot 2019-03-28 at 12 13 10" src="https://user-images.githubusercontent.com/16057093/55145114-308a0d80-5153-11e9-853d-30d06f9a2a2e.png">

- **Freckle Automation**
<img width="1440" alt="Screen Shot 2019-03-28 at 12 12 51" src="https://user-images.githubusercontent.com/16057093/55145150-439cdd80-5153-11e9-8d36-ca96c05e1256.png">
